task1/longGenerated.sh

```diff
@@ -0,0 +1,123 @@
+W_bPxA=Ha@q{Qdnf3&:E,$Rx_uKm'$kz$w9a1$04XY_n$2b`p			D !QJ'avaF%gGx$TF62t$igwvuc7T$VayKl2k3$FlsA$m8g$0PNx0$8$YNcAFZF7'$DTwFukWS		 2\
+)i
+a
+
+ 	$fvT1A4$SnG	Xqb$Oa''$9nLQ
+	K
+,	& $jLsibKkL$piqjbv$CFFZa
+  @9^b&		d	 nB'4x?yHx`1F$v91DMhJ
+IEfLXt='$vJU_4Utg0
+
+ Rl$N]x 	$Ma
+
+	GS''$gRJXETn'$Nj8wPW35fo:,,.}vXbSxd72]a<$MtF$d1e$Oq$u7xfFrAF4]*}H7Qpk'^}PD		 ''$bfdnWx1''$X$ygCTPv |)
+
+ k , yj
+p<g			p
+''
+Z	s
+
+Af 	>l
+
+
+C`ZyO`''
+
+	Dz
+0
+\F
+$s_t_7t4I$G|
+{	 	 $eXBA
+b=
+	''$WKh0O$vkE70 $Z1w8X3I'
+wFDcy='$64JvIG$gYV3ol''U:
+
+	  $5zs_IXY
+
+
+	$hun6o$g7f_vao''  		p$q_m$kt$pVczwzkLF  S
+ q ''$eM8WZ9
+	MAC6$639CKJqj		+
+
+#
+	$TrxRyYa >)/
+ @	I('
+JVZRxVwa='$J335E''		m
+ $I c<
+H
+  9u$j$5t''$9xy44p4n'
+iFztwnvD=' $UVXI5cQK3
+
+h
+	1	/
+*+
+''		 	2
+Rm	$IyFJp$zD$LVc''
+`2A N
+
+ 		u
+
+Fv!	 D|'
+KdwB='$ZcGb$jf
+B
+bq	n		''T(
+ $C7HHB	n
+$mE
+i;
+k$rFSr7zJ''$qtlOGvM6$Jv$r4rR	i
+$$4CF6A8m$henrL	''$IOm0u$NTOz4wHZL$G4jc4bzcR`	? h/n	@:dv$Wvc3UXk
+ U+lp
+''$k7KrO$SqBiP t}
+	  M 	U#j$8MWmje4rP$UJvCzA''N
+		 	Hc
+-$RKCedh$nDIGzKWNG:Dz	PT  $ctdVTmz''	/$9a4mTE$oNG
+
+W+$S3nc
+9Ges			 6n
+	D0
+	V_/''
+$Q	$3BHKJcD$h$f$KYwGnKpQF'']	T0i'
+qPdVGpQj='#A
+
+
+$Ifh3wyub9*
+2t 1m''		W
+ 		$ILSFOzQ :
+
+ $21 H a|$P46xluOCY''$WFyKxx''$C7u_mytO  <	 $kYfh$SS
+Y		:	  9	 F
+@Ob<khx
+
+	c''$AkiY:
+gn
+I ss=
+/ zyo
+-]N8F	[vF x
+A
+
+	w 	f
+
+
+tC 	o	:l
+
+		# `v 5I& ''$eDgdvZ$3Nh7 ='Q]:[oGvMZ
+yqJtBBWq=$rM3O$TSKKO$DR$2Pqe$HVTIyT$WALGiYZ$G0Ol'$Hv4Q$T892sgtzDt
+$UiO6TNlW OQ$h_VFPdxp$aF'-f3;rg(->V7F7{3$O7Senq8b(WB[W!+$yM55lUi'$Vkri`
```

Comment:
@sashasashasasha151 plz update this line correspondingly to new rules regarding brackets in string literals. At the moment this script failed to be parsed due to
```
    |
104 | $UiO6TNlW OQ$h_VFPdxp$aF'-f3;rg(->V7F7{3$O7Senq8b(WB[W!+$yM55lUi'$Vkri`
    |                                ^
unexpected '('
expecting end of input or expression
```